etc/profile-a-l/anki.profile

```diff
@@ -57,15 +62,23 @@ novideo
 protocol unix,inet,inet6
 # QtWebengine needs chroot to set up its own sandbox
 seccomp !chroot
+seccomp.block-secondary
 
 disable-mnt
-private-bin anki,mplayer,mpv,python*
+# env is required for python scripts on Gentoo Linux.
+# anki uses mpv or mplayer for playing audio and uses lame to record audio.
+# sh and dbus-send are used by aqt/theme.py to query dark mode through
+# org.freedesktop.portal.Desktop.
+private-bin anki,dbus-send,env,lame,lua*,mplayer,mpv,python*,sh
```

Comment:
Is `private-bin lua*` actually needed?

mpv uses libluajit directly.

Reply:
I've been lazy. I will get back soon.

Reply:
> I've been lazy. I will get back soon.

That's alright; it might be helpful to focus on one PR at a time.

I'd leave this and maybe 2 other PRs opened and close the rest if you don't
plan on working on them soon (and then they could gradually be reopened as the
open PRs are merged and as work is started on the closed ones).

This would also make reviews easier.

Reply:
It's been a while; I'll merge it without the lua item.

If you find that it makes a difference, let me know.